src/System.Numerics.Tensors/src/Properties/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("System.Numerics.Tensors.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100039ac461fa5c82c7dd2557400c4fd4e9dcdf7ac47e3d572548c04cd4673e004916610f4ea5cbf86f2b1ca1cb824f2a7b3976afecfcf4eb72d9a899aa6786effa10c30399e6580ed848231fec48374e41b3acf8811931343fc2f73acf72dae745adbcb7063cc4b50550618383202875223fc75401351cd89c44bf9b50e7fa3796")]
```

Comment:
I thought we were going to remove this?  What internals are still necessary for the tests?

Reply:
Because the various operations and arithmetic support requires some of the internal utilities. There is also a lot of code that currently is accessing the fields directly, rather than going through the properties.

Reply:
I assumed the whole reason for moving the operations and arithmetic code to the unit tests was to ensure that someone could write those operations themselves if they needed to.

See https://github.com/dotnet/corefxlab/pull/2578#discussion_r230524052
and
https://github.com/dotnet/corefxlab/issues/1798#issuecomment-336947110

Reply:
Yes, and they still can if we finish cleaning up the templates to use the public properties, rather than the internal fields and they write their own equivalents to some of our array utility functions. However, we need to get the package up first and can finish the remaining code cleanup after the fact.